src/sentry/silo.py

```diff
@@ -9,34 +9,38 @@
 from django.conf import settings
 
 
-class ServerComponentMode(Enum):
-    """Defines the component mode of the application to be acting as."""
+class SiloMode(Enum):
+    """Defines which "silo" component the application is acting as.
+
+    The default choice is "monolith", which assumes that the server is the only
+    "silo" in its environment and allows access to all tables and endpoints.
+    """
 
     MONOLITH = "MONOLITH"
     CONTROL = "CONTROL"
     CUSTOMER = "CUSTOMER"
     FRONTEND = "FRONTEND"
```

Comment:
When would we need the frontend silo?

Reply:
Oh right, that was some early base-covering in case we needed a Django component to serve the small handful of pages that are Django views (for auth and stuff) alongside what's serving the static webpack resources. But now it seems safe to assume that would be handled by the customer silos. I'll take it out.